lib/hanami/middleware/body_parser/class_interface.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'errors'
+require_relative 'parser'
+
+module Hanami
+  module Middleware
+    class BodyParser
+      module ClassInterface
+        # @since x.x.x
```

Comment:
Not sure what we're mean to be putting for these `@since` doc tags

Reply:
Once is release it will have the version that it was introduced. Hope, I'm not wrong @jodosha